src/store/personViews.js

```diff
@@ -226,11 +289,118 @@ export default function personViews(state = null, action) {
         });
     }
     else {
-        return state || {
-            viewList: createList(),
-            columnsByView: {},
-            matchesByViewAndQuery: {},
-            rowsByView: {},
-        };
+        // Not a view related action, but it could still be an action that
+        // affects data in a view, and should flag a row as dirty. Check
+        // for action types which should have this affect, and create a list
+        // of dirtyPersonIds in dirty views.
+        const dirtyPersonIds = [];
+        const affectedViewIds = [];
+
+        if (action.type == types.ADD_TAGS_TO_PERSON + '_FULFILLED') {
```

Comment:
I suppose we could add more types here later, e.g. person data or custom fields?

Reply:
Yes exactly, as hinted in the PR description. 👍